cluster/kubernetes/manifests.go

```diff
@@ -7,12 +7,36 @@ import (
 	"github.com/weaveworks/flux/resource"
 )
 
+// namespacer assigns namespaces to manifests that need it (or "" if
+// the manifest should not have a namespace.
+type namespacer interface {
+	correctNamespace(kresource.KubeManifest) error
+}
+
+// Manifests is an implementation Manifests particular to
```

Comment:
_an implementation_ **of cluster.**_Manifests_?

Reply:
I briefly considered blaming golint for forcing me into the awkward grammar but that would be a deflection